lte/gateway/c/core/oai/include/state_manager.h

```diff
@@ -165,21 +168,77 @@ class StateManager {
         is_initialized,
         "StateManager init() function should be called to initialize state");
 
+    auto start = std::chrono::high_resolution_clock::now();
     std::string proto_str;
     ProtoUe ue_proto = ProtoUe();
     StateConverter::ue_to_proto(ue_context, &ue_proto);
     redis_client->serialize(ue_proto, proto_str);
     std::size_t new_hash = std::hash<std::string>{}(proto_str);
-
+    auto stop            = std::chrono::high_resolution_clock::now();
+    std::cout << "TIME PROTOBUF CONVERSION : "
```

Comment:
I would suggest enclosing all benchmarking code that is added on a CMake boolean flag, similar to the one we're using for MME unit testing: https://sourcegraph.com/github.com/magma/magma/-/blob/lte/gateway/c/core/oai/tasks/mme_app/mme_app_context.c?L733:6 

This will help to keep current implementation code (without benchmark code) separate, what do you think?

Reply:
Yes, I propose EXPERIMENTAL_TEST or BENCHMARK_TEST ? Please suggest.